apim-operator/README.md

```diff
@@ -0,0 +1,38 @@
+#### Developer Guide
+
+**Building the image**
+
+The apim-operator was built using the operator-sdk CLI tool. you can follow the [user guide][operator_sdk_user_guide] of operator-sdk CLI tool for 
```

Comment:
Let's use the first letter capital when starting the sentence, eg: You can follow ..

Reply:
Resolved